rdkit/build.sh

```diff
@@ -21,12 +24,13 @@ cmake \
     -D CMAKE_BUILD_TYPE=Release \
     .
 
-make -j$CPU_COUNT
 
 if [[ `uname` == 'Linux' ]]; then
+    make -j$CPU_COUNT 
     RDBASE=$SRC_DIR LD_LIBRARY_PATH="$PREFIX/lib:$SRC_DIR/lib" PYTHONPATH=$SRC_DIR ctest -j$CPU_COUNT --output-on-failure
     RDBASE=$SRC_DIR LD_LIBRARY_PATH="$PREFIX/lib:$SRC_DIR/lib" PYTHONPATH=$SRC_DIR $PYTHON "$RECIPE_DIR/pkg_version.py"
 else
+    make -j$CPU_COUNT install
```

Comment:
this 'make install' in the osx branch looks a bit odd to me, I guess there's probably a reason for introducing it, but I'll comment anyway, just in case it's an oversight.

Reply:
I put that in there when I was in the flailing "why the hell doesn't this work?" stage caused by the new conda python builds.
I'll try taking it out to see if things still work without it.